yarn.lock

```diff
@@ -4573,6 +4573,11 @@ lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.5:
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.19.tgz#e48ddedbe30b3321783c5b4301fbd353bc1e4a4b"
   integrity sha512-JNvd8XER9GQX0v2qJgsaN/mzFCNA5BRe/j8JN9d+tWyGLSodKQHKFicdwNYzWwI3wjRnaKPsGj1XkBjx/F96DQ==
 
+lodash@^4.17.21:
+  version "4.17.21"
```

Comment:
I _think_ you can consolidate this with the above chunk if you remove both chunks then run `yarn`.

Reply:
I'm going to pass on making this change as editing `yarn.lock` manually generally leads to more problems than it's worth in my experience!